src/webapp/pages/form-page/FormPage.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { useParams } from "react-router-dom";
+
+import i18n from "../../../utils/i18n";
+import { DiseaseOutbreakEventForm } from "./disease-outbreak-event/DiseaseOutbreakEventForm";
+
+export type FormType = "disease-outbreak-event";
+
+export const FormPage: React.FC = React.memo(() => {
+    const { formType, id } = useParams<{
+        formType: FormType;
+        id?: string;
+    }>();
+
+    switch (formType) {
+        case "disease-outbreak-event":
+            return <DiseaseOutbreakEventForm diseaseOutbreakEventId={id} formType={formType} />;
```

Comment:
Maybe this is part of the temporal code, but in which case is useful id to be undefined?

Reply:
Because if the id is undefined it means that it's create and not edit, and instead of load the values we should have an empty form

Reply:
Ok, that's what I thought. So, take this is a general note, do not change it here: with TS, the safest way to express the idea "new form or edit form with ID" is with a union: `{type: "new"} | {type: "edit", id: Id}`, which could be for example a prop of a component.

Reply:
Noted! :pray: